tests/system/test_system.py

```diff
@@ -289,6 +289,46 @@ def test_rollback_on_connection_closing(self):
         cursor.close()
         conn.close()
 
+    def test_execute_many(self):
+        # connect to the test database
+        conn = Connection(Config.INSTANCE, self._db)
+        cursor = conn.cursor()
+
+        cursor.execute(
+            """
+INSERT INTO contacts (contact_id, first_name, last_name, email)
+VALUES (1, 'first-name', 'last-name', 'test.email@domen.ru'),
+       (2, 'first-name2', 'last-name2', 'test.email2@domen.ru')
```

Comment:
```suggestion
VALUES (1, 'first-name', 'last-name', 'test.email@example.com'),
       (2, 'first-name2', 'last-name2', 'test.email2@example.com')
```

Not a big deal, but we probably want to avoid including non-google, non-example domains anywhere, even in test code.